relay-common/src/time.rs

```diff
@@ -130,7 +130,8 @@ impl UnixTimestamp {
 
     /// Returns the timestamp as chrono datetime.
     pub fn as_datetime(self) -> Option<DateTime<Utc>> {
-        NaiveDateTime::from_timestamp_opt(self.0 as i64, 0).map(|n| DateTime::from_utc(n, Utc))
+        NaiveDateTime::from_timestamp_opt(self.0 as i64, 0)
+            .map(|n| DateTime::from_naive_utc_and_offset(n, Utc))
```

Comment:
`from_utc` is [deprecated](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.from_utc) since chrono 0.4.27. The function does the same [under the hood](https://docs.rs/chrono/0.4.28/src/chrono/datetime/mod.rs.html#123-136).